internal/website/content/howto/runtimevar/_index.md

```diff
@@ -173,6 +173,37 @@ variable.
 [`awsparamstore.OpenVariable`]:
 https://godoc.org/gocloud.dev/runtimevar/awsparamstore#OpenVariable
 
+### AWS Secrets Manager {#awssm}
+
+To open a variable stored in [AWS Secrets Manager][] via a URL, you can use the
+`runtimevar.OpenVariable` function as shown in the example below.
+
+[AWS Secrets Manager]:
+https://aws.amazon.com/secrets-manager
+
+`runtimevar.OpenVariable` will create a default AWS Session with the
+`SharedConfigEnable` option enabled; if you have authenticated with the AWS CLI,
+it will use those credentials. See [AWS Session][] to learn about authentication
+alternatives, including using environment variables.
+
+[AWS Session]: https://docs.aws.amazon.com/sdk-for-go/api/aws/session/
+
+{{< goexample
+"gocloud.dev/runtimevar/awssecretsmanager.Example_openVariableFromURL" >}}
+
+#### AWS Constructor {#awssm-ctor}
+
+The [`awssecretsmanager.OpenVariable`][] constructor opens a Secrets Manager
+variable.
+
+{{< goexample "gocloud.dev/runtimevar/awssecretsmanager.ExampleOpenVariable" >}}
```

Comment:
In order for these examples to work, you need to annotate them in the code.

Update https://github.com/google/go-cloud/blob/master/runtimevar/awssecretsmanager/example_test.go to look similar to https://github.com/google/go-cloud/blob/master/runtimevar/awsparamstore/example_test.go (with the PRAGMA comments).

Reply:
Done.